share/tools/web_config/sample_prompts/sorin.fish

```diff
@@ -0,0 +1,60 @@
+# Name:   Sorin
+# Author: Ivan Tham <ivanthamjunhoe@gmail.com>
+
+# If login shell, don't do anything
+if status --is-login; and function fish_prompt; echo -n '% '; end; end
```

Comment:
Then shouldn't that be tested via "test $TERM = Linux"?

Login shells can also be used in other contexts, and there can also be non-login shells in a framebuffer VT.